src/algorithms/calorimetry/CalorimeterHitsMerger.cc

```diff
@@ -88,7 +88,7 @@ void CalorimeterHitsMerger::init() {
 
   // loop through provided readout fields
   auto& svc = algorithms::ServiceSvc::instance();
-  for (std::size_t iField = 0; std::string & field : fields) {
+  for (std::size_t iField = 0; std::string& field : fields) {
```

Comment:
`ReferenceAlignment: Left` in clang-format 16 does not seem to affect range-based for with initialization.